app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -123,61 +110,59 @@ class ReceiveModal extends React.PureComponent {
     }
 
     return (
-      <Modal onClose={closeReceiveModal}>
-        <Box width={9 / 16} mx="auto">
-          <Heading.h1 textAlign="center">
-            <WalletName wallet={activeWalletSettings} />
-            {networkInfo && networkInfo.id === 'testnet' && networkInfo.name}
-          </Heading.h1>
-
-          <Bar pt={2} />
-
-          <Flex justifyContent="center" my={3}>
-            <Tabs onClick={this.setQrcode} activeKey={qrCodeType} items={tabs} />
-          </Flex>
-
-          <Flex mt={4} mb={3} alignItems="center" flexDirection="column">
-            {qrCode && qrCodeType !== QRCODE_TYPE_LNDCONNECT && (
-              <QRCode value={qrCode} size="xlarge" />
-            )}
-            {qrCodeType === QRCODE_TYPE_LNDCONNECT && (
-              <>
-                <QRCode
-                  value={qrCode}
-                  size="xlarge"
-                  obfuscate={qrCodeType === QRCODE_TYPE_LNDCONNECT && !reveal}
+      <Box width={9 / 16} mx="auto">
```

Comment:
Rather than hard coding the width and margin onto this component, it would be better to apply `{...rest}` to the wrapping `Box`, and have these props passed in by the caller.

Reply:
I didn't hardcode them (this is how it was), but your suggestion makes sense

Reply:
updated